src/script/tracking/EventTrackingRepository.ts

```diff
@@ -202,7 +202,6 @@ export class EventTrackingRepository {
   }
 
   private unsubscribeFromProductTrackingEvents(): void {
-    amplify.unsubscribeAll(WebAppEvents.ANALYTICS.SUPER_PROPERTY);
```

Comment:
We were never subscribing to this and `SUPER_PROPERTY` was [removed](https://github.com/wireapp/wire-web-packages/pull/3140).